selection/dbms/postgres_dbms.py

```diff
@@ -84,8 +84,10 @@ def indexes_size(self):
         return result[0]
 
     def drop_database(self, database_name):
-        self.exec_only("drop database {}".format(database_name))
-        logging.info("Database {} dropped".format(database_name))
+        statement = f"""DROP DATABASE {database_name};"""
```

Comment:
Use same style for exec and logging:
`self.exec_only( f"DROP DATABASE {database_name};")`